### PR TITLE
Lesson 1 and 2 Task 6 done

### PR DESCRIPTION
**Pull request by iamipheco**
@promise-J
@ChidiPatrick
@OgJaykay
@Pieta9221
@MiniHack
@Melumm

I'm done with my Lesson 1 & 2 Task 4, Please Kindly Check it out.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                <option value="nigeria">Nigeria</option>
+                <option value="ghana">Ghana</option>
+                <option value="cameroon">Cameroon</option>
+                <option value="uk">UK</option>
+                <option value="usa">USA</option>
+                <option value="germany">Germany</option>
+            </select>
+            
+            <label for="terms">I agree to the terms and conditions:</label>
+            <input type="checkbox" id="terms" name="terms" required>
```
Comment (ChidiPatrick) [inline docs/index.html]: Close your input elements at lines 47 and 49. <input/>

**Review by ChidiPatrick — CHANGES_REQUESTED** [review]
Nice work. Please close all your input elements properly

File: docs/index.html (hunk 1 above)
Comment (iamipheco) [reply docs/index.html]: > Close your input elements at lines 47 and 49.

All corrections have been made 
what is the next now

**Review by ChidiPatrick — APPROVED** [review]
Certified, you can now merge

**Comment by iamipheco** [comment]
> Certified, you can now merge

Alright Thanks

**Review by ChidiPatrick — COMMENTED** [review]
Great! You can now merge

**Review by OgJaykay — COMMENTED** [review]
Good work👍👍👍.You can merge now

**Pull request merged**